base/math.jl

```diff
@@ -146,6 +146,10 @@ sqrt(x::Float32) = box(Float32,sqrt_llvm(unbox(Float32,x)))
 sqrt(x::Real) = sqrt(float(x))
 @vectorize_1arg Number sqrt
 
+# 1-arg hypot
+hypot(x::Real) = abs(promote(float(x))...)
```

Comment:
Sorry I meant for line 160 not 150. No reason really (I just thought to be consistent with the three arguments version)